### ISSUE TITLE
terminal errors on open

### ISSUE THREAD

**Issue by johnpapa** [Microsoft/vscode#59183]
Issue Type: <b>Bug</b>

When I show the bottom panel to get to the terminal, it briefly opens and then closes with an error.

This started happening on my mac today after the update to the new insiders and updating extensions. Not sure what is causing it. I just re-opened code insiders and disabled all extensions and the error is still happening.

```
The terminal process command '/usr/local/bin/zsh -l' failed to launch (exit code: 1)
```

The other bottom panels will open fine (e.g problems, debug output). But when switching to terminal it closes again.

![2018-09-23_08-48-27](https://user-images.githubusercontent.com/1202528/45928214-279f5180-bf0e-11e8-9093-28422b3a29b4.gif)

VS Code version: Code - Insiders 1.28.0-insider (2558a72d6902015b142ef45c57b664ce1f09e03f, 2018-09-21T05:12:46.853Z)
OS version: Darwin x64 17.7.0

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-4980HQ CPU @ 2.80GHz (8 x 2800)|
|GPU Status|2d_canvas: enabled<br>checker_imaging: disabled_off<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: enabled<br>rasterization: enabled<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|3, 3, 3|
|Memory (System)|16.00GB (4.12GB free)|
|Process Argv|/Applications/Visual Studio Code - Insiders.app/Contents/MacOS/Electron -psn_0_1241391|
|Screen Reader|no|
|VM|0%|

</details><details><summary>Extensions (26)</summary>

Extension|Author (truncated)|Version
---|---|---
ng-template|Ang|0.1.10
azds|azu|0.1.120180911
markdown-preview-github-styles|bie|0.1.3
path-intellisense|chr|1.4.2
vscode-markdownlint|Dav|0.20.0
vscode-eslint|dba|1.6.0
EditorConfig|Edi|0.12.4
tslint|eg2|1.0.39
vscode-npm-script|eg2|0.3.5
prettier-vscode|esb|1.6.1
Angular1|joh|0.2.0
angular-essentials|joh|0.4.0
Angular2|joh|6.1.5
vscode-azureappservice|ms-|0.9.1
vscode-azurefunctions|ms-|0.10.2
vscode-cosmosdb|ms-|0.8.0
azure-account|ms-|0.4.3
azurecli|ms-|0.4.2
debugger-for-chrome|msj|4.10.1
angular2-inline|nat|0.0.17
vetur|oct|0.12.6
vscode-docker|Pet|0.3.0
material-icon-theme|PKi|3.6.0
polacode|pnp|0.2.2
vue-vscode-snippets|sdr|1.3.0
code-settings-sync|Sha|3.1.2

(1 theme extensions excluded)

</details>

**Comment by johnpapa** [comment]
I *think* i resolved this ... I had these settings in code:

```
  "terminal.integrated.shell.osx": "/usr/local/bin/zsh",
  "terminal.integrated.shell.linux": "/usr/local/bin/zsh",
```

I've had these for a long time. I ran `where zsh` and it found `/bin/zsh`. 

When I remove these 2 lines, the integrated terminal works now.

I then checked and the default setting for these are indeed `/bin/zsh`

Not sure why this just became a problem, but this resolved it for me.

Hope this helps someone else.
